internal/gen/types/typescript/typescript.go

```diff
@@ -47,6 +48,9 @@ func Run(ctx context.Context, projectId string, dbConfig pgconn.Config, schemas
 		if strings.Contains(utils.Config.Api.Image, "v9") {
 			postgrestV9Compat = true
 		}
+
+		// Use custom network when connecting to local database
+		networkID = utils.NetId
```

Comment:
```suggestion
		dbConfig.Host = utils.DbAliases[0]
		dbConfig.Port = 5432
		networkID = utils.NetId
```

Yup, host and port need to be updated as well.